code/game/machinery/harvester.dm

```diff
@@ -83,9 +83,7 @@
 	if(!occupant || !iscarbon(occupant))
 		return
 	var/mob/living/carbon/C = occupant
-	operation_order = list() //we're pretty much inverting C.bodyparts here, since we want chest and head last, wich are otherwise the first to go
-	for(var/i in LAZYLEN(C.bodyparts) to 1 step -1)
-		operation_order += C.bodyparts[i]
+	operation_order = reverseList(C.bodyparts)   //Chest and head are first in bodyparts, so we invert it to make them suffer more
```

Comment:
this reverses the list in place which is why I said to take a copy

C.bodyparts.Copy()

otherwise you might have unintended issues

Reply:
reverseList already copies it

Reply:
I totally forgot I asked you to do that lol

smoothbrain